hc/front/tests/test_add_check.py

```diff
@@ -11,4 +11,14 @@ def test_it_works(self):
         self.assertRedirects(r, "/checks/")
         assert Check.objects.count() == 1
 
-    ### Test that team access works
+    def test_team_access_works(self):
+        url = "/checks/add/"
+        self.client.login(username="alice@example.org", password="password")
+        self.client.post(url)
+        check = Check.objects.filter()
+        check_code = list(check)[0].code
```

Comment:
Why did you convert check into a list ? 
Is check a tuple ? 
if so, whats the problem of accessing the  check[0].code values for the tuple